client/components/common/ProfilePictureModal.tsx

```diff
@@ -0,0 +1,162 @@
+/** @jsx jsx */
+
+import React, { useEffect, useState } from 'react';
+import { jsx } from '@emotion/core';
+import { Modal, notification } from 'antd';
+import { useRouter } from 'next/router';
+
+import { useTranslation } from 'i18n';
+import { getImageUrl } from 'common/utils';
+import { PROFILE_PICTURES, mq } from '../../common/constants';
+import { gray7, blue6 } from 'common/mixins';
+import changeProfilePictureMutation from 'graphql/mutations/changeProfilePicture.graphql';
+import {
+  changeProfilePicture,
+  changeProfilePictureVariables,
+} from 'graphql/mutations/__generated__/changeProfilePicture';
+import { useMutation } from '@apollo/client';
+import userProfileQuery from 'graphql/queries/userProfile.graphql';
+import buildListQuery from 'graphql/queries/buildList.graphql';
+
+interface Props {
+  username: string;
+  visible: boolean;
+  onCancel: () => void;
+  currentlyActive: string;
+}
+
+const ProfilePictureModal: React.FC<Props> = ({
+  username,
+  onCancel,
+  visible,
+  currentlyActive,
+}) => {
+  const { t } = useTranslation('common');
+  const [active, setActive] = useState<any>('');
```

Comment:
Why aren't you able to type this?

Reply:
Was experimenting with which value I actually wanted to use, ended up with string and forgot to change typing haha.